scripts/utils/import-metrics.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Script using mimirtool for prometheus tsdb collection
+#
+# Usage: ./import-metrics.sh path/to/kubeconfig.yaml
+#
+# Important: Must set inputs for selector and time range
+# 
+# For all metrics use selector='{__name__!=""}'
+# Time format YYYY-MM-DDThh:mm:ssZ  ex: "2025-01-31T00:00:00Z"
+# bash helpers: 
+#      date -u +"%Y-%m-%dT%H:%M:%SZ"
+#      date -u -v-3H +"%Y-%m-%dT%H:%M:%SZ"
+#      [-v[+|-]val[y|m|w|d|H|M|S]]
+selector='{__name__!=""}' 
```

Comment:
Can we make `selector`, `from` and `to` into script args with defaults instead of requiring users to manually enter them into the script? It would be good to add comments to each input to clarify to users what the defaults are set to

Reply:
Done! added better descriptions, added a function to the process args and set input variables